pkg/metastore/mock/metastore_client.go

```diff
@@ -66,11 +73,16 @@ func (m *MSClient) GetPartition(_ context.Context, dbName string, tableName stri
 	return m.partitionMap[GetPartitionKey(dbName, tableName, values)], nil
 }
 
-func (m *MSClient) GetDatabase(_ context.Context, _ string) (*metastore.Database, error) {
-	panic("implement me")
+func (m *MSClient) GetDatabase(_ context.Context, name string) (*metastore.Database, error) {
+	m.t.Helper()
```

Comment:
why do you need this one? usually or mock just simulate something that is tested by the main test flow which does the assert.
this we should drop this one and make sure we can check the state using the mock and not let it produce helpers - unless it is a validate method that the mock provides and that we pass 'testing.T' and not keeping it as member.